### ISSUE TITLE
Ability to change the weight

### ISSUE THREAD

**Issue by tuxtlequino** [valentine195/obsidian-leaflet-plugin#236]
This is more noticeable with high resolution monitors, but not being able to determine the weight can result in having a little preview window. Since height can be determine as a percentage of the available space, if this is changed to percentage, the end result is a long skinny window. 

Would it be possible to also have an option for the weight?

Thank you for creating and maintaining this plugin. It is truly awesome.

**Comment by valentine195** [comment]
Width?

the maps at the moment should be 100% of the available width, is that not the case for you?

**Comment by tuxtlequino** [comment]
No. It is 100% till you expand the preview window to a certain point. But like I said, with high resolution monitors or large monitors, that width can be small. It occurred to me that it may be related to some responsive CSS attributes, but I couldn't figure it out. Out of my depth.

**Comment by valentine195** [comment]
Interesting, yeah, the map’s container is just set to `width: 100%`, which will make it take up 100% of the map’s container’s container. It is likely something is clamping that width somehow. 

I’m on my phone but maybe try this: 
If you open the dev console (Ctrl-shift-i/cmd-alt-i on Mac), I believe you’re looking for the `.markdown-preview-view` or `.markdown-preview-sizer`. Clicking on them will reveal their Styles, clicking on the Computed tab will let you see where the width is coming from. 

If we can’t find it, I’ll look tomorrow when I’m on my computer!

**Comment by tuxtlequino** [comment]
.markdown-preview-view.is-readable-line-width .markdown-preview-sizer {
    max-width: 700px;
    margin-left: auto;
    margin-right: auto;
}

There was the problem. There is a max width of 700px. I just changed that to another value.

**Comment by valentine195** [comment]
Ah, great find! You could also turn off readable line width in Settings > Editor. Didn’t think of that!

**Comment by tuxtlequino** [comment]
Thank you!
